app/electron/main.js

```diff
@@ -167,17 +167,22 @@ autoUpdater.on('update-downloaded', () => {
 });
 
 app.on('ready', createWindow);
-app.on('window-all-closed', function () {
+
+app.on('quit', function () {
   if (serverProcess) {
     serverProcess.stdin.pause();
     serverProcess.kill();
   }
-  if (process.platform !== 'darwin') {
```

Comment:
Maybe we want to set the serverProcess to `null` after killing it above? So if the signal happens twice for some reason we don't call kill twice.